src/main/res/values-fr/string.xml

```diff
@@ -195,11 +195,11 @@
 
     <string name="action_menu_post_title">Action sur le post</string>
 
-    <string name="action_upvote">Upvote</string>
-    <string name="action_downvote">Downvote</string>
+    <string name="action_upvote">Vote positif</string>
```

Comment:
Are the words "upvote" and "downvote" not used on the French reddit? They seem like reddit-specific words, rather than English words that need translating. I'm not the expert on that though :)

Reply:
I don't know what the other languages do. It's a bit strange to use english wording whereas there is a french translation for a positive vote or a negative one.

https://fr.bab.la/dictionnaire/anglais-francais/upvote